protocol-dashboard/src/components/UserName/UserName.tsx

```diff
@@ -27,6 +29,9 @@ const UserName = ({ className, wallet, hasLoaded }: UserNameProps) => {
         })}
       />
       {!!name && (nonWalletName ? name : formatShortWallet(wallet))}
+      {!!data && (
```

Comment:
Looks like you were just being consistent with the line before but let's prefer a ternary with an explicit `null` here